plugin.video.ad6/resources/lib/README.md

```diff
@@ -0,0 +1,7 @@
+# Readme

```

Comment:
not needed as well (sample markdown file)

Reply:
Removed